src/components/Header/Header.module.css

```diff
@@ -1,44 +1,86 @@
 .header {
   display: flex;
+  padding: 24px 16px;
   align-items: center;
   justify-content: space-between;
-  padding: 22px 80px;
   height: 100px;
+  position: relative;
+}
+
+.sticky {
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 200;
+  background-color: var(--bg-light);
+}
+
+@media screen and (min-width: 1194px) {
+  .header {
+    padding: 24px 48px;
+  }
+}
+
+@media screen and (min-width: 1920px) {
+  .header {
+    padding: 22px 80px;
+  }
 }
 
 .logo {
-  width: 200px;
-  height: 72px;
+  width: 132px;
+  height: 48px;
+}
+
+@media screen and (min-width: 1194px) {
+  .logo {
+    width: 154px;
+    height: 56px;
+  }
 }
 
 .headerMenu {
-  display: flex;
-  justify-content: space-between;
+  display: none;
+}
+
+@media screen and (min-width: 1194px) {
+  .headerMenu {
+    display: flex;
+    justify-content: space-between;
+  }
 }
```

Comment:
Why you don't write in one @media all styles for one resolution ? Just combine all styles in one request